torchvision/models/detection/ssd.py

```diff
@@ -0,0 +1,486 @@
+import torch
+import torch.nn.functional as F
+import warnings
+
+from collections import OrderedDict
+from torch import nn, Tensor
+from typing import Any, Dict, List, Optional, Tuple
+
+from . import _utils as det_utils
+from .anchor_utils import DBoxGenerator
+from .backbone_utils import _validate_trainable_layers
+from .transform import GeneralizedRCNNTransform
+from .. import vgg
+from ..utils import load_state_dict_from_url
+from ...ops import boxes as box_ops
+
+__all__ = ['SSD', 'ssd300_vgg16']
+
+model_urls = {
+    'ssd300_vgg16_coco': None,  # TODO: Add url with weights
+}
+
+backbone_urls = {
+    # We port the features of a VGG16 backbone trained by amdegroot because unlike the one on TorchVision, it uses the
+    # same input standardization method as the paper. Ref: https://s3.amazonaws.com/amdegroot-models/vgg16_reducedfc.pth
+    'vgg16_features': 'https://download.pytorch.org/models/vgg16_features-amdegroot.pth'
```

Comment:
Hum... this is not really ideal... Let's chat about this on a call

Reply:
255 * weights of Fist conv VS 255 * on first Module. TBD